llmfoundry/command_utils/eval.py

```diff
@@ -78,18 +75,6 @@ def evaluate_model(
                     'parallelism_config cannot contain deprecated fsdp_config arguments.',
                 )
 
-    if fsdp_config:
```

Comment:
could be remembering this wrong, were there some reg tests which had some issues with this? i think it should be ok but just want to make sure